docs/guides/setup/configuring-data-contexts/how-to-create-a-new-data-context-with-the-cli.md

```diff
@@ -2,4 +2,11 @@
 title: How to configure a new Data Context with the CLI
 ---
 
-This article is a stub.
+We recommend that you create new Data Contexts by using the a ``greate_expectations init`` command in the directory where you want to deploy Great Expectations.
+
+```bash
+great_expectations init
+```
+
+Please see the [Getting Started](../../../tutorials/getting-started/initialize-a-data-context.md) section for a more detailed tutorial on Data Contexts and getting a new project up and running.
+
```

Comment:
This one is a bit odd -- do we not want to put in our standard getting started, with the JavaScript import -- even if this is not much content?

Reply:
Which JS import are you referring to?